src/main/java/org/isf/medicalstock/manager/MovBrowserManager.java

```diff
@@ -122,4 +151,53 @@ private void check(LocalDateTime from, LocalDateTime to, String errMsgKey) throw
 			}
 		}
 	}
+
+	/**
+	 * Get the last Movement.
+	 *
+	 * @return the retrieved movement.
+	 * @throws OHServiceException
+	 */
+	public Movement getLastMovement() throws OHServiceException {
+		return ioOperations.getLastMovement();
+	}
+
+	/**
+	 * Deletes the last Movement.
+	 *
+	 * @param lastMovement - the last movement to delete
+	 * @throws OHServiceException
+	 */
+	@Transactional
+	public void deleteLastMovement(Movement lastMovement) throws OHServiceException {
+
+		MovementType movType = medicalDsrStockMovTypeManager.getMovementType(lastMovement.getType().getCode());
+		String lotCode = lastMovement.getLot().getCode();
+		Medical medical = lastMovement.getMedical();
+		int medicalCode = medical.getCode();
+		int quantity = lastMovement.getQuantity();
+		if (movType.getType().equals("+")) {
```

Comment:
We had problems with some charset with "+", please use `contains`

```suggestion
		if (movType.getType().`equals`("+")) {
```